packages/retail-ui-extensions/src/components/List/List.ts

```diff
@@ -19,6 +42,7 @@ export interface ListRowLeftSide {
      * A link to an image to be displayed on the far left side of the row.
      */
     source: string;
+    badge?: number;
```

Comment:
A comment here to describe what the badge does would be helpful

Reply:
Yeah I missed it and was hoping I could push it before anyone noticed whoops